requirements/main.txt

```diff
@@ -1,14 +1,8 @@
-numpy==1.19.0
-scipy==1.5.0
-statsmodels==0.11.1
-pmdarima==1.6.1
-matplotlib==3.2.2
-fbprophet==0.6
-pandas==1.1.0
-torch==1.5.1
-tensorboard==2.2.2
-ipython==7.15.0
-tqdm==4.46.1
-holidays==0.10.2
-plotly==4.9.0
-ipywidgets==7.5.1
\ No newline at end of file
+numpy>=1.18.1
+scipy>=1.4.1
+statsmodels>=0.11.1
+matplotlib>=3.2.1
+pandas>=0.23.1
+ipython>=7.14.0
+tqdm>=4.46.0
+holidays>=0.10.2
```

Comment:
@pennfranc Should I use strict version numbers or allow greater or equal versions?

Reply:
We switched to equal versions recently!